### ISSUE TITLE
Everywhere except the menu bar, the font does not follow systemwide settings 

### ISSUE THREAD

**Issue by Anixx** [otya128/winevdm#183]
The default font is System, even if the font set systemwide is different.

It does not follow the smoothing and color settings of the system.

**Comment by Anixx** [comment]
This is a screenshot from 32-bit version of Taipei Mahjongg:
![clipboard_20190218 2](https://user-images.githubusercontent.com/1271736/52976175-19783300-33d1-11e9-9652-69dbf3c0e5e9.png)
As you can see, it uses system font in the about box.

**Comment by cracyc** [comment]
This is part of the design of the dialog.  You can open the program in the Borland Resource Workshop and change the fonts manually but IMO winevdm should use whatever font the dialog is intended to use which is System by default.

**Comment by Anixx** [comment]
Is there a way to substitute another font for system so that it affected only Win16 apps?

**Comment by cracyc** [comment]
Nope.  The dialog converter can be forced to use a different font but the results are not good so I'm not going to submit a change to do it.

**Comment by Anixx** [comment]
I see. It would be interesting to test this version though (or maybe this can be made an option?).

**Comment by Anixx** [comment]
The current approach is not perfect either:

![clipboard_20190218](https://user-images.githubusercontent.com/1271736/52971167-24c26300-33bf-11e9-9ae5-cdbc1714ef22.png) 
(look at the buttons)
